bigbluebutton-html5/imports/ui/components/tooltip/component.jsx

```diff
@@ -39,24 +36,16 @@ class Tooltip extends Component {
       dynamicTitle: this.dynamicTitle,
       delay: this.delay,
       onShow: this.onShow,
-      onHide: this.onHide,
+      touchHold: true,
     };
 
     this.tooltip = Tippy(`#${this.tippySelectorId}`, options);
   }
 
   onShow() {
-    document.addEventListener('keyup', this.handleEscapeHide);
-  }
-
-  onHide() {
-    document.removeEventListener('keyup', this.handleEscapeHide);
-  }
-
-  handleEscapeHide(e) {
-    if (e.keyCode !== ESCAPE) return;
-
-    this.tooltip.tooltips[0].hide();
+    window.setTimeout(() => {
```

Comment:
Where is the requirement to hide the tooltip after 5s?

Reply:
The flash client's tool tips are hidden after a given time has elapsed. I was unsure the exact time in seconds used by the flash client, it was suggested that 5 seconds would suffice.

Also, hiding the tooltip after X amount of time fixes an overlap that occurs when the client is viewed in a desktop browser that is scaled to a small size.
![desktop-tooltip-overlaping](https://user-images.githubusercontent.com/22058534/34833709-149f1c3c-f6be-11e7-97e3-690792039ba4.jpg)

Reply:
We needed because thats how WAI-ARIA describe the tooltip interaction with keyboard. 

> **Keyboard Interaction**
Escape: Dismisses the Tooltip.

https://www.w3.org/TR/wai-aria-practices-1.1/#tooltip

Reply:
I still miss the point of having a timer since the tooltip should hide after the element lose focus, in a real scenario is someone having something focused for more than 5s? Having the tooltip on this time a real problem?

:man_shrugging: I dont really know. @tylercopeland probably can help us with those questions.

Reply:
@oswaldoacauan @KDSBrowne great questions here. My recommendation would be to remove tooltips when a participant is using a mobile device. Anything that we’re use to doing on a desktop that involves a hover, isn’t the best experience when we try to replicate it on the mobile device. Also, if we look at trends for mobile experiences, tooltips aren’t typically used.